README.md

```diff
@@ -39,8 +39,50 @@ For further details, see our [Installing ArcLight](https://github.com/sul-dlss/a
 
 ## Usage
 
+Arclight is a Ruby gem designed to work with archival data. It can be installed on a server or virtual server and runs at the usual ruby web address. Once running, finding aids in the form of archival collection data can be imported into Arclight through an indexing process. Institutional and repositories data can also be added to Arclight (Currently this requires a developer. Configuration pages will be added for this in future versions). Additional finding aids can be added at any time.
+
+After data indexing, Arclight can to be used to search, browse, and display the repositories (sets of collections), collections, and components within collections. Globally available search allows filtering on several types of terms (Keyword, Name, Place, etc.). Once a search is begun, it can be further narrowed using facets on the left side of the search page. Selecting a search result goes directly to that results show or display page. Also global available are buttons for Repositories and Collections which can be used an any time.
+
+Browsing allows you to view the Overview or Contents (when it exists) of a collection. The Overview tab displays top level metadata about the collection. The Contents tab displays an outline view of a next level of the collection. You can expand each level by selecting (clicking). Selecting a component in the Contents views goes to a component page which shows the metadata for it.
+
+Some pages include an inline view tab to the right of an item which will expand the Contents further.
+
 See the [ArcLight demo](https://arclight-demo.projectblacklight.org/) and [ArcLight MVP Wiki](https://github.com/sul-dlss/arclight/wiki) for usage.
 
+**Arclight Major Features**
```

Comment:
Could we move this section to a new wiki page while I'm thinking about it and link to it in the previous line?